perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -149,8 +158,21 @@ def _IsReady(self):
     Returns:
       True if the resource was ready in time, False if the wait timed out.
     """
+    cmd = util.GcloudCommand(self, 'sql', 'instances', 'describe',
+                             self.instance_id)
+    stdout, _, _ = cmd.Issue()
+    json_output = json.loads(stdout)
```

Comment:
a few lines above, for the _Exists() method, we used the retcode of gcloud describe being 0 to imply the resource exists, but that is fragile, it's possible the return code is for the execution status of the glcoud command itself, not for the output of the gcloud. If the resource does not exist but gcloud successfully determined it does not exist, it might just return 0 and output an error message on command line.

I'd consider using a different way of detecting existence. Please look at the AWS side of implementation to see what should be used.

@gareth-ferneyhough fyi